fe/package.json

```diff
@@ -27,6 +27,7 @@
     "bs-platform": "^4.0.7",
     "css-loader": "^1.0.1",
     "file-loader": "^2.0.0",
+    "font-awesome": "^4.7.0",
```

Comment:
@ktrzos 
Jeśli chodzi o font-awesome to ja do index.html dodalem linka: <link rel="stylesheet" ... 
Użyłem to tylko w jednym miejscu: LoadingMask - to jest ten kręciołek.
Twój sposób z importem jest lepszy, więc jak coś aby font-awesome się nie dupklikował to możesz wywalić to z index.html i pewnie trzeba obsłużyć w LoadingMask przez import

Reply:
Ok, zrobione. Przy okazji okazało się, że korzystałem z nieaktualnego pakietu, który wspierany był do wersji 4.x (a mamy 5.x). Również i to zaktualizowałem, a następnie logikę FA przeniosłem do globalnego *.scss. Spójrz proszę czy jest ok.